app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
+    app.register_blueprint(descriptor_blueprint, url_prefix='/descriptor')
+
+    from resource import resource as resource_blueprint
+    app.register_blueprint(resource_blueprint, url_prefix='/resource')
+
+    from suggestion import suggestion as suggestion_blueprint
```

Comment:
For now I think it's the latter, because it seems to me that all external users' actions (viewing the map & suggestions) should be in one blueprint, `external_user` or something like that. But I guess we don't have to make that decisions right now anyway.